libr/core/cmd.c

```diff
@@ -1456,14 +1456,13 @@ static int cmd_kuery(void *data, const char *input) {
 			}
 			cur_cmd = r_str_ndup (out, cur_pos - out);
 
-			r_cons_printf ("\n\n\"%s\" : [", cur_cmd);
+			r_cons_printf ("\n\"%s\" : [", cur_cmd);
 
 			next_cmd = r_str_newf ("anal/%s/*", cur_cmd);
 			temp_storage = sdb_querys (s, NULL, 0, next_cmd);
 
 			if (!temp_storage) {
-				r_cons_println ("\nEMPTY\n");
-				r_cons_printf ("],\n\n");
+				r_cons_printf ("EMPTY],");
```

Comment:
EMPTY is not a valid json statement

Reply:
just return an empty array `[]` (but using the `pj` api reusing the same instance, is not a good practice to use printf to construct json objects.. actually this `!temp_storage` must become an `if (temp_storage) {` to wrap the loop below. so the empty array object will be built by pj without an extra work.